src/audiomixerboard.cpp

```diff
@@ -1098,24 +1137,78 @@ void CAudioMixerBoard::ChangeFaderOrder ( const EChSortType eChSortType )
 
 void CAudioMixerBoard::UpdateTitle()
 {
-    QString strTitlePrefix = "";
+    // if the server runs --mastermix and we are not the master (channelID 0), disable all faders
+    if ( eMasterMixState == MM_ENABLED && iMyChannelID != 0 )
+    {
+        DisableFaders();
+    }
+    else
+    {
+        EnableFaders();
+    }
 
+    QList<QString> title_parts;
+    
     if ( eRecorderState == RS_RECORDING )
     {
-        strTitlePrefix = "[" + tr ( "RECORDING ACTIVE" ) + "] ";
+        title_parts.append("[" + tr ( "RECORDING ACTIVE" ) + "] ");
     }
 
-    setTitle ( strTitlePrefix + tr ( "Personal Mix at: " ) + strServerName );
+    if ( eMasterMixState == MM_ENABLED )
+    {
+        if ( iMyChannelID == 0 )
+        {
+            title_parts.append( "YOU CONTROL: " );
+        }
+        
+        title_parts.append("Server Mix ");
+    }
+    else
+    {
+        title_parts.append("Personal Mix ");
+    }
+    
+    title_parts.append("at: ");
+    title_parts.append(strServerName);
+    setTitle ( title_parts.join("") );
     setAccessibleName ( title() );
 }
 
+void CAudioMixerBoard::DisableFaders()
+{
+    for ( int i = 0; i < MAX_NUM_CHANNELS; i++ )
+    {
+        if ( i == 0 || i == iMyChannelID )
+        {
+            // Do not disable the master's our own fader in the mix, so we can control those
+            continue;
+        }
```

Comment:
I blame these lines of code for a bug discovered in yesterday's rehearsal: 
**Some people were not able to control their own fader** (the only enabled fader was the mix master's fader).

I noticed that the affected people's IPs showed up as "connected" twice in the server log:

```
2021-05-05 19:11:57, 94.223.115.113, connected (3) <-- connected once
2021-05-05 19:12:39, 94.223.115.113, connected (4) <-- connected twice
[...]
2021-05-05 19:16:01, 85.212.238.106, connected (12) <-- connected once
[...]
2021-05-05 19:19:49, 85.212.238.106, connected (18) <-- connected twice
```

Choir colleague `94.223.115.113` reported that she connected via WiFi initally and then plugged in Ethernet and turned of WiFi afterwards. 

I assume this leaves the the `CAudioMixerBoard` in a somehow inconsistent state with `iMyChannelID`.

Reply:
To do:

- [ ] Reproduce
- [ ] Fix